mysite/fsm/mixins.py

```diff
@@ -247,11 +260,9 @@ def get_message(self, chat, current=None, message=None):
                 userMessage=True,
                 is_additional=is_additional,
             )
-            if not self.fsm.fsm_name_is_one_of('live_chat'):
-                message = Message.objects.get_or_create(**_data)[0]
-            else:
-                message = Message(**_data)
-                message.save()
+            # here was Message.objects.create for all fsm's except love_chat. for live_chat fsm here was get_or_create
```

Comment:
`love_chat` :)

Reply:
yeah love chat.... we are working on Love chat :)